src/libraries/System.Diagnostics.DiagnosticSource/tests/ActivitySourceTests.cs

```diff
@@ -360,6 +360,40 @@ public void TestActivityCreationProperties()
                 }
             }).Dispose();
         }
+
+        [Fact]
+        public void TestDefaultParentContext()
+        {
+            RemoteExecutor.Invoke(() => {
+                using (ActivitySource aSource = new ActivitySource("ParentContext"))
+                {
+                    using (ActivityListener listener = new ActivityListener
+                    {
+                        ShouldListenTo = (activitySource) => activitySource.Name == "ParentContext",
+                        GetRequestedDataUsingContext = (ref ActivityCreationOptions<ActivityContext> activityOptions) =>
+                        {
+                            Activity c = Activity.Current;
+                            if (c != null)
+                            {
+                                Assert.Equal(c.Context, activityOptions.Parent);
+                            }
+
+                            return ActivityDataRequest.AllData;
+                        }
```

Comment:
FWIW, I find the use of property initializers for something in a using statement makes this difficult to read, as it's visually difficult to distinguish them from the body of the using block.  You might consider using the new C# 8 using statement that doesn't use braces and also not using property initializers in order to make the code less indented and easier to read.

Reply:
I am going to address this in a separate PR as I want to catch Preview 6 with this change.